src/Parallelization.php

```diff
@@ -516,4 +517,36 @@ private function runTolerantSingleCommand(
             }
         }
     }
+    
+/**
+     * @param InputInterface $input
+     * @param string[] $blackListParams
+     * @return string[]
+     */
+    private function serializeInputOptions(InputInterface $input, array $blackListParams) : array {
+        $options = array_diff_key(
```

Comment:
can you make sure that `array_diff_key`, `array_filter ` and `array_fill_keys` are properly imported? (via a `use function` statement)

Reply:
added